bitnami/cilium/values.yaml

```diff
@@ -0,0 +1,1515 @@
+      ##
+      honorLabels: false
+      ## @param operator.metrics.serviceMonitor.interval Interval at which metrics should be scraped.
+      ## ref: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#endpoint
+      ## e.g:
+      ## interval: 10s
+      ##
+      interval: ""
+      ## @param operator.metrics.serviceMonitor.scrapeTimeout Timeout after which the scrape is ended
+      ## ref: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#endpoint
+      ## e.g:
+      ## scrapeTimeout: 10s
+      ##
+      scrapeTimeout: ""
+      ## @param operator.metrics.serviceMonitor.metricRelabelings Specify additional relabeling of metrics
+      ##
+      metricRelabelings: []
+      ## @param operator.metrics.serviceMonitor.relabelings Specify general relabeling
+      ##
+      relabelings: []
+      ## @param operator.metrics.serviceMonitor.selector Prometheus instance selector labels
+      ## ref: https://github.com/bitnami/charts/tree/main/bitnami/prometheus-operator#prometheus-configuration
+      ## selector:
+      ##   prometheus: my-prometheus
+      ##
+      selector: {}
+
+## @section Other Parameters
+##
+
+## RBAC configuration
+##
+rbac:
+  ## @param rbac.create Specifies whether RBAC resources should be created
+  ##
+  create: true
+  ## @param rbac.rules Custom RBAC rules to set
+  ## e.g:
+  ## rules:
+  ##   - apiGroups:
+  ##       - ""
+  ##     resources:
+  ##       - pods
+  ##     verbs:
+  ##       - get
+  ##       - list
+  ##
+  rules: []
+
+## @section KeyValue Store Parameters
+##
+
+## etcd chart configuration
+## ref: https://github.com/bitnami/charts/blob/main/bitnami/etcd/values.yaml
+##
+etcd:
+  ## @param etcd.enabled Deploy etcd to satisfy the KeyValue Store requirements
+  ## To use an external kvstore set this to false and configure the `externalKvstore.*` parameters
+  ##
+  enabled: false
+  ## @param etcd.replicaCount Number of etcd replicas
+  ##
+  replicaCount: 1
+  ## @param etcd.auth.rbac.create Switch to enable RBAC authentication
+  ##
+  auth:
+    rbac:
+      create: false
+  ## @param etcd.resourcesPreset Set container resources according to one common preset (allowed values: none, nano, small, medium, large, xlarge, 2xlarge). This is ignored if resources is set (resources is recommended for production).
+  ## More information: https://github.com/bitnami/charts/blob/main/bitnami/common/templates/_resources.tpl#L15
+  ##
+  resourcesPreset: "micro"
+  ## @param etcd.resources Set container requests and limits for different resources like CPU or memory (essential for production workloads)
+  ## Example:
+  ## resources:
+  ##   requests:
+  ##     cpu: 2
+  ##     memory: 512Mi
+  ##   limits:
+  ##     cpu: 3
+  ##     memory: 1024Mi
+  ##
+  resources: {}
+  ## @param etcd.service.ports.client etcd client port
+  ##
+  service:
+    ports:
+      client: 2379
+## External KeyValue Store Configuration
+##
+externalKvstore:
```

Comment:
In other charts we use `externalEtcd`, any reason for using `externalKvstore`? Is it because there are other supported kv stores?

Reply:
Exactly. You can use other kv stores.